games-roguelike/adom/adom-3.3.3.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
didn't use `repoman commit` ? ;)

Reply:
pkgcheck scan --commits / pkgdev commit would handle this too. Don't commit manually, use either pkgdev or repoman.

Reply:
ok let's make a deal - how about I go ahead and set up some git hooks that runs `pkgcheck --scan --commits` prior to pushing anything.

I *want* to want to use `pkgdev commit`, but it truly just does not feel good to me.

Reply:
That works for me, but I recommend you run pkgcheck scan --commits before you're ready to push ;)

Reply:
>  I recommend you run pkgcheck scan --commits before you're ready to push ;)

Maybe I'll add it as a pre-commit hook then :)